how-to/operations/removing-the-primary-node.rst

```diff
@@ -67,6 +67,16 @@ Remove the ``k8s`` snap:
 
    sudo snap remove --purge k8s
 
+Remove the ``microovn`` snap:
+
+.. note ::
+
+   The above steps should be run only if the node is configured as a network node.
```

Comment:
we don't really need an admonishment for that.

We don't say it for the hypervisor snap that it must be when it's compute.